### PR TITLE
Preparations for v249

### PR DESCRIPTION
**Pull request by bluca**

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -3,7 +3,8 @@ systemd System and Service Manager
 CHANGES WITH 249 in spe:
```
Comment (keszybz) [inline NEWS]: Please also drop "in spe" here.

File: NEWS [hunk 2]
```diff
@@ -479,6 +480,9 @@ CHANGES WITH 249 in spe:
           storage directly (if enabled) and it will avoid analyzing the core file
           to print backtrace and metadata in the journal.
 
+        * coredumpctl gained a new 'help' verb to bring it in line with other
+          tools.
```
Comment (keszybz) [inline NEWS]: I don't think we want to advertise this. `help` is not documented in the man page or `--help` output. We provide it, but don't advertise.

**Review by keszybz — CHANGES_REQUESTED** [review]

File: NEWS (hunk 2 above)
Comment (bluca) [reply NEWS]: removed

File: NEWS (hunk 1 above)
Comment (bluca) [reply NEWS]: done

**Comment by bluca** [comment]
Also added a fixup for the whitespace formatting of https://github.com/systemd/systemd/issues/20125

**Comment by poettering** [comment]
lgtm

**Comment by keszybz** [comment]
bionic-i386 has passed, let's merge.

**Pull request merged**